Fix #10 Created a new endpoint for getting all hobbies of all participants

delClg opened the pull request:
### Create new endpoint for getting all hobbies of all participants #10 
Created a new endpoint which returns an array with the hobbies of ALL the participants in profiles.json, where no hobby is mentioned twice
Added a link to the new endpoint in the homepage public/index.html
<img width="507" alt="1" src="https://user-images.githubusercontent.com/96474794/157908831-85522a11-2aff-41c5-bf72-6f2ac82336c9.png">
<img width="682" alt="1" src="https://user-images.githubusercontent.com/96474794/157911583-2ff84dbc-0a8f-47e7-9a14-d31a39c096d7.png">

N-Shar-ma commented:
@delClg A couple issues here:
- Your screenshot does not match the index.html file you changed
- Though the code works, having hobbies in a roll-number variable isn't really sensible. Think of a different approach within the time limit.

delClg commented:
Sorry ma'am, I linked the wrong image. My image folder is really cluttered. I corrected it now.
Also, I found out about a new function called flatMap in Java and it works in JavaScript too! I also added a new router GET for the /hobbies
